sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateClient.cs

```diff
@@ -29,30 +29,30 @@ public class CertificateClient
         private const string ContactsPath = "/contacts/";
 
         /// <summary>
-        /// Protected constructor for mocking
+        /// Initializes a new instance of the <see cref="CertificateClient"/> class for mocking.
         /// </summary>
         protected CertificateClient()
         {
 
         }
 
         /// <summary>
-        /// Creates a new certificate client for the specified vault.
+        /// Initializes a new instance of the <see cref="CertificateClient"/> class for the specified vault.
         /// </summary>
-        /// <param name="vaultUri">The vault on which the certificate client operates</param>
-        /// <param name="credential">The credential used to authenticate requests to the vault</param>
+        /// <param name="vaultUri">A <see cref="Uri"/> to the vault on which the client operates.</param>
```

Comment:
Since this is called DnsName in the portal do we want to call out that it might also be referred to as such?

Reply:
Good idea.